include/engine/guidance/assemble_leg.hpp

```diff
@@ -38,8 +36,7 @@ RouteLeg assembleLeg(const DataFacadeT &facade,
     auto distance = std::accumulate(leg_geometry.segment_distances.begin(),
                                     leg_geometry.segment_distances.end(), 0.);
     auto duration = std::accumulate(route_data.begin(), route_data.end(), 0.,
-                                    [](const double sum, const PathData &data)
-                                    {
+                                    [](const double sum, const PathData &data) {
```

Comment:
Maybe. Since yesterday @MoKob and I can reproduce identical formatting (for the record: with LLVM 3.8).